### PR TITLE
ROECCT-133-beneficial-owners-involved

### PR DESCRIPTION
**Pull request by ndeekonda-ch**
### JIRA link

https://companieshouse.atlassian.net/jira/software/c/projects/ROECCT/boards/562?selectedIssue=ROECCT-133

### Change description
Beneficial owners involved in overseas entity - add radio buttons

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: views/includes/page-templates/relevant-period-statements-add-radios.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    legend: {
+    isPageHeading: true,
+    classes: "govuk-fieldset__legend--xl"
+    }
+    },
+    items: [
+    {
+    value: "individualOwner",
+    text: "Individual beneficial owner",
+    checked: individualOwner === individualOwner
```
Comment (ConnorOS-ch) [inline views/includes/page-templates/relevant-period-statements-add-radios.html:21]: checked isn't required her, we only need to use checked if we want the radio buttons to be pre-populated when a user navigates to the page

File: views/update/update-beneficial-owner-type.html [hunk 2]
```diff
@@ -94,7 +94,10 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
     <form method="post">
       {% include "includes/csrf_token.html" %}
       {% include "includes/beneficial-owner-type-add-radios.html" %}
-
+      {% if update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD" %}
+      <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
```
Comment (ConnorOS-ch) [inline views/update/update-beneficial-owner-type.html:98]: Can you add some unit tests to `benifical.owner.type.spec` to check these extra radio buttons are rendered. you can pass the mock `UPDATE_OBJECT_MOCK_RELEVANT_PERIOD_CHANGE` in `session.mock` to mock the first checkbox being ticked.

**Review by ConnorOS-ch — COMMENTED** [review]
Looking good so far

**Review by ConnorOS-ch — APPROVED** [review]

File: views/update/update-beneficial-owner-type.html [hunk 3]
```diff
@@ -94,7 +94,10 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
     <form method="post">
       {% include "includes/csrf_token.html" %}
       {% include "includes/beneficial-owner-type-add-radios.html" %}
-
+      {% if update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD" %}
```
Comment (ConnorOS-ch) [inline views/update/update-beneficial-owner-type.html:97]: Can you add `feature_flag_relevant_period === true` to this if statement to protect from unwanted behavior

File: src/controllers/update/beneficial.owner.type.controller.ts [hunk 4]
```diff
@@ -54,7 +56,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       templateName: config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE,
       ...appData,
       hasExistingBosMos,
-      hasNewlyAddedBosMos
+      hasNewlyAddedBosMos,
+      dateOfCreation: getRegistrationDate(appData.update?.date_of_creation as InputDate)
```
Comment (ConnorOS-ch) [inline src/controllers/update/beneficial.owner.type.controller.ts:60]: You will need to pass in your feature flag here:
`feature_flag_relevant_period: config.FEATURE_FLAG_ENABLE_RELEVANT_PERIOD`

**Review by ConnorOS-ch — COMMENTED** [review]
A couple of small changes to add the feature flag to the new radio buttons

**Comment by ndeekonda-ch** [comment]
closed this pr due to refractor the code and open new below one 
https://github.com/companieshouse/overseas-entities-web/pull/1430

**Pull request closed without merging**